src/SubHask/Category.hs

```diff
@@ -130,7 +130,7 @@ instance Category (->) where
     type ValidCategory (->) (a :: *) = ()
     id = P.id
 
-    {-# NOINLINE (.) #-}
+    {-# INLINE (.) #-}
```

Comment:
I don't know why this was NOINLINE, but i need it to be INLINE for Fusion-Rules to apply..

Could this trigger a regression in other parts or dependend libraries (like HLearn)?